public/js/pimcore/asset/listfolder.js

```diff
@@ -459,6 +459,25 @@ pimcore.asset.listfolder = Class.create(pimcore.asset.helpers.gridTabAbstract, {
         return config;
     },
 
+    onRawDeleteSelectedRows: function () {
+        var ids = [];
```

Comment:
```suggestion
        let ids = [];
```
another issue i see is that all new `var` should be either `let`/`const`

Reply:
var, let, and const are used to declare variables, each with different scope, hoisting, and reassignment behaviors:

var: Function or globally scoped, hoisted, can be redeclared and reassigned. Generally avoided in modern JS due to less predictable scope.
let: Block-scoped, hoisted but not accessible until declared, can be reassigned but not redeclared in the same scope. Preferred for variables that change.
const: Block-scoped, hoisted but not accessible until declared, cannot be reassigned or redeclared. Use for variables that should not change after initialization.
Usage Guidelines:

Avoid var in favor of let and const to prevent scope-related bugs.
Use let for variables that need to be reassigned.
Default to const for variables that should not change, enhancing code readability and safety.